is-compatible/action.yml

```diff
@@ -27,7 +27,7 @@ runs:
     - id: run-levitate
       run: |
         # RUN levitate is-compatible. Save the output to .levitate_output
-        npx --yes @grafana/levitate@latest is-compatible --path ${{ inputs.module }} --target ${{ inputs.targets }} --markdown | tee .levitate_output
+        npx --yes @grafana/levitate@latest is-compatible --path ${{ inputs.module }} --target ${{ inputs.targets }} --markdown | tee .levitate_output || true
```

Comment:
Does it also mean that the following line will always evaluate to be "zero"?

Reply:
Yes. I am pushing a fix for this. I will rely on the command text output rather than the exit code for this check.